src/org/apache/xerces/jaxp/datatype/XMLGregorianCalendarImpl.java

```diff
@@ -48,19 +48,18 @@
  * defined in the XML Namespace
  * <code>"http://www.w3.org/2001/XMLSchema"</code>.
  * These datatypes are normatively defined in 
- * <a href="http://www.w3.org/TR/xmlschema-2/#dateTime">W3C XML Schema 1.0 Part 2, Section 3.2.7-14</a>.</p>
+ * <a href="http://www.w3.org/TR/xmlschema-2/#dateTime">W3C XML Schema 1.0 Part 2, Section 3.2.7-14</a>.
  *
- * <p>The table below defines the mapping between XML Schema 1.0
- * date/time datatype fields and this class' fields. It also summarizes
- * the value constraints for the date and time fields defined in
- * <a href="http://www.w3.org/TR/xmlschema-2/#isoformats">W3C XML Schema 1.0 Part 2, Appendix D,
- * <i>ISO 8601 Date and Time Formats</i></a>.</p>
+ * <p>The table below defines the mapping between XML Schema 1.0 date/time datatype fields and this class' fields.
+ * It also summarizes the value constraints for the date and time fields defined in
+ * <a href="http://www.w3.org/TR/xmlschema-2/#isoformats">W3C XML Schema 1.0 Part 2, Appendix D, <i>ISO 8601 Date and Time Formats</i></a>.
+ * </p>
  * 
- * <a name="datetimefieldsmapping"/>
- * <table border="2" rules="all" cellpadding="2">
+ * <a name="datetimefieldsmapping"></a>
+ * <table border="1">
  *   <thead>
  *     <tr>
- *       <th align="center" colspan="3">
+ *       <th colspan="3">
```

Comment:
align

Reply:
putting in `style="text-align: center;"` as that's the better way to do this now

Reply:
that's fine, but it doesn't seem to be here yet. Do you need to push?